waf_support.go

```diff
@@ -0,0 +1,73 @@
+package waf
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-multierror"
+)
+
+var wafSupportErrors []error
+var wafManuallyDisabledErr error
```

Comment:
Worth a comment explaining how do they must be used

Reply:
```suggestion
// Errors used to report data using the Health function
// Store all the errors related to why go-liddwaf is unavailable for the current target at runtime.
var wafSupportErrors []error
// Not nil if the build tag `datadog.no_waf` is set
var wafManuallyDisabledErr error
```